ci/ci_run_all_plugin_tests.sh

```diff
@@ -37,6 +37,7 @@ set -e
 mv __pysys_junit_xml pysys_junit_xml_apt
 
 sudo cp ../../plugins/tedge_docker_plugin/tedge_docker_plugin.sh /etc/tedge/sm-plugins/docker
+sudo chmod +x /etc/tedge/sm-plugins/docker
```

Comment:
This kind of ad hoc install makes me think we are missing an install script from the source tree.

Reply:
See #712